usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/DateTime.java

```diff
@@ -0,0 +1,216 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+
+import org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException;
+
+public class DateTime implements Comparable<DateTime> {
+    private static final int HOUR_START_POS = 11;
+    private static final int DAYS_30 = 30;
+    private static final int DAYS_31 = 31;
+    private static final int FEBRUARY = 2;
+    private static final int JULY = 7;
+    private static final int LEAP_YEAR_FREQ = 4;
+    private static final int TWO_LAST_DIGITS = 100;
+    private static final int DAYS_29 = 29;
+    private static final int DAYS_28 = 28;
+    private static final int DECEMBER = 12;
+    private static final int MAX_HOUR = 24;
+    private static final int MAX_MINUTES_SECONDS = 60;
+    private static final int FIRST_TWO_DIGITS = 10;
+    private static final int MONTH_START_POS = 5;
+    private static final int DAY_START_POS = 8;
+    private static final int HOUR_END_POS = 13;
+    private static final int MIN_START_POS = 14;
+    private static final int MIN_END_POS = 16;
+    private static final int SEC_START_POS = 17;
+    private int year;
+    private int month;
+    private int day;
+    private int hour;
+    private int minutes;
+    private int seconds;
+
+    public DateTime(){}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['}' is not preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5314325)